<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>486</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>755</width>
    <height>486</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>755</width>
    <height>486</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>750</width>
     <height>465</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>750</width>
     <height>465</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Sales</string>
     </attribute>
     <widget class="QPushButton" name="addSale">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Sale</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserID">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>User ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="Product">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Product</string>
      </property>
     </widget>
     <widget class="QLabel" name="SalesPrice">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>385</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sales Price</string>
      </property>
     </widget>
     <widget class="QLabel" name="Amount">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Quantity</string>
      </property>
     </widget>
     <widget class="QLabel" name="Date">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>350</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>       Date</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="UserIDText">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>350</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="DateText">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>350</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="SalesPriceText">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>390</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="AmountText">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>350</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="ProductText">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>390</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="newLog">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>New Log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addLog">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>380</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveLog">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>410</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Log</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="SaleLogs">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>331</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>141</number>
      </attribute>
      <column>
       <property name="text">
        <string>User ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Product</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sales Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Members</string>
     </attribute>
     <widget class="QTableWidget" name="UserLogs">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>711</width>
        <height>331</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>118</number>
      </attribute>
      <column>
       <property name="text">
        <string>User ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total Sales</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Rebate</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Expires</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prefered?</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="UserID_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>User ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="Name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="Expires">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>340</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Expires</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="NameText">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>390</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="ExpiresText">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>350</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="UserIDText_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>350</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="isPrefered">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>390</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Prefered Member?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addUser">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add User</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addProfile">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>380</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Profile</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newProfile">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>New Profile</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveProfile">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>410</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Profile</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuBy_Month">
     <property name="title">
      <string>By Month</string>
     </property>
     <addaction name="actionJanuary"/>
     <addaction name="actionFebruary"/>
     <addaction name="actionMarch"/>
     <addaction name="actionApril"/>
     <addaction name="actionMay"/>
     <addaction name="actionJune"/>
     <addaction name="actionJuly"/>
     <addaction name="actionAugust"/>
     <addaction name="actionSeptember"/>
     <addaction name="actionOctober"/>
     <addaction name="actionNovember"/>
     <addaction name="actionDecember"/>
    </widget>
    <addaction name="menuBy_Month"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionSales_Log"/>
     <addaction name="actionMember_Profile"/>
    </widget>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionSales_Log_2"/>
     <addaction name="actionMember_Profile_2"/>
    </widget>
    <widget class="QMenu" name="menuSave">
     <property name="title">
      <string>Save</string>
     </property>
     <addaction name="actionSales_Log_3"/>
     <addaction name="actionMember_Profile_3"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="menuOpen"/>
    <addaction name="menuSave"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilter"/>
  </widget>
  <action name="actionJanuary">
   <property name="text">
    <string>January</string>
   </property>
  </action>
  <action name="actionFebruary">
   <property name="text">
    <string>February</string>
   </property>
  </action>
  <action name="actionMarch">
   <property name="text">
    <string>March</string>
   </property>
  </action>
  <action name="actionApril">
   <property name="text">
    <string>April</string>
   </property>
  </action>
  <action name="actionMay">
   <property name="text">
    <string>May</string>
   </property>
  </action>
  <action name="actionJune">
   <property name="text">
    <string>June</string>
   </property>
  </action>
  <action name="actionJuly">
   <property name="text">
    <string>July</string>
   </property>
  </action>
  <action name="actionAugust">
   <property name="text">
    <string>August</string>
   </property>
  </action>
  <action name="actionSeptember">
   <property name="text">
    <string>September</string>
   </property>
  </action>
  <action name="actionOctober">
   <property name="text">
    <string>October</string>
   </property>
  </action>
  <action name="actionNovember">
   <property name="text">
    <string>November</string>
   </property>
  </action>
  <action name="actionDecember">
   <property name="text">
    <string>December</string>
   </property>
  </action>
  <action name="actionSales_Log">
   <property name="text">
    <string>Sales Log</string>
   </property>
  </action>
  <action name="actionMember_Profile">
   <property name="text">
    <string>Member Profile</string>
   </property>
  </action>
  <action name="actionSales_Log_2">
   <property name="text">
    <string>Sales Log</string>
   </property>
  </action>
  <action name="actionMember_Profile_2">
   <property name="text">
    <string>Member Profile</string>
   </property>
  </action>
  <action name="actionSales_Log_3">
   <property name="text">
    <string>Sales Log</string>
   </property>
  </action>
  <action name="actionMember_Profile_3">
   <property name="text">
    <string>Member Profile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
